src/System.Private.ServiceModel/src/System/ServiceModel/Description/XmlSerializerOperationBehavior.cs

```diff
@@ -412,6 +412,12 @@ private void CreateHeaderDescriptionTable(MessageDescription message, MessageInf
                         else if (headersMapping != null)
                         {
                             XmlMemberMapping memberMapping = headersMapping[headerNameIndex++];
+#if FEATURE_NETNATIVE
```

Comment:
Interesting question -- I think we found RuntimeInformation was not supported in NET Native yet.  If it is, it would be very useful for places like this.